features/gezag-minderjarige/2a.1-zijn-juridische-ouders-nu-met-elkaar-gehuwd-of-partners.feature

```diff
@@ -51,12 +51,17 @@ Functionaliteit: 2a.1 - Zijn juridische ouders nu met elkaar gehuwd of partners
       zijn.
 
   Gebruikte velden:
-    - BSN ouder 1 -> 02.01.20 
-    - BSN ouder 2 -> 03.01.20
-    - BSN in huwelijk/geregistreerd partnerschap -> 05.01.20
-    - Datum huwelijkssluiting/aangaan geregistreerd partnerschap -> 05.06.10
-    - Datum ontbinding huwelijk/geregistreerd partnerschap -> 05.07.10
-    - Reden huwelijksontbinding -> 05.07.40
+    - Geboortedatum persoon -> 01.03.10
+    - Burgerservicenummer ouder 1 -> 02.01.20
+    - Geslachtsnaam ouder 1 -> 02.02.40
+    - Burgerservicenummer ouder 2 -> 03.01.20
+    - Geslachtsnaam ouder 2 -> 03.02.40
+    - Burgerservicenummer huwelijk/geregistreerd partnerschap -> 05.01.20
+    - Start datum huwelijk/geregistreerd partnerschap -> 05.06.10
+    - Eind datum huwelijk/geregistreerd partnerschap -> 05.07.10
+    - Eind reden huwelijk/geregistreerd partnerschap -> 05.07.40
+    - Datum opschorting bijhouding -> 07.67.10
+    - Reden opschorting bijhouding -> 07.67.20
```

Comment:
```suggestion
    - Datum opschorting bijhouding (PL ouder 1) -> 07.67.10
    - Datum opschorting bijhouding (PL ouder 2) -> 07.67.10
    - Reden opschorting bijhouding (PL ouder 1) -> 07.67.20
    - Reden opschorting bijhouding (PL ouder 2) -> 07.67.20
```

Zo beter?

Reply:
vind ik beter inderdaad let wel dat ouder 2 niet 07 is

Reply:
Besproken. Akkoord.

Reply:
Deze aanpassing is op meerdere plekken doorgevoerd om consistent te blijven. Zie eb89ec95e46cc83770f651e9057f2b1b1f5b184c.